app/delegate.go

```diff
@@ -33,6 +33,7 @@ func (parentModel *model) newItemDelegate(keys *keyMap) list.DefaultDelegate {
 			fullValue = i.TitleFull()
 			fp = i.FilePath()
 			desc = i.TimeStamp()
+
 		} else {
```

Comment:
I would have removed the else by return when `!ok`

Reply:
@ccoVeille ah yes I missed this one. Thanks for pointing that out, updating it now :rocket: